src/main/java/org/apache/platypus/server/utils/TarImpl.java

```diff
@@ -126,4 +145,30 @@ private static void addFilestoTarGz(String filePath, String parent, TarArchiveOu
             }
         }
     }
+
+    public static void main(String[] args) throws IOException {
```

Comment:
Would it be better to have this in tests (even though it doesn't test with asserts) ?

Reply:
moved to TarImplTest